skopt/tests/test_space.py

```diff
@@ -415,3 +415,25 @@ def test_real_distance():
     reals = Real(1, 10)
     for i in range(1, 10+1):
         assert_equal(reals.distance(4.1234, i), abs(4.1234 - i))
+
+
+@pytest.mark.parametrize("space, name",
+                         [(Real(1, 2, name="learning rate"), "learning rate"),
+                          (Integer(1, 100, name="no of trees"), "no of trees"),
+                          (Categorical(["red, blue"], name="colors"), "colors")])
+def test_dimension_name(space, name):
```

Comment:
I think `space` should be replaced with `dimension`?

Reply:
nice catch!